src/components/SearchResults.js

```diff
@@ -1,5 +1,6 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import moment from "moment";
+import CustomerProfile from "./CustomerProfile";
 
```

Comment:
This file is a bit big, it should be broken into smaller files with one component each